my_gazebo_concert/src/solutions/gazebo.services

```diff
@@ -0,0 +1,13 @@
+# PRIORITIES
+#
+# Turtle pond has scheduler_msgs.Request.DEFAULT_PRIORITY, teleop has 
+# scheduler_msgs.Request.HIGH_PRIORITY. If you want to change the priority
+# override it as below for any of the services:
+#  overrides:
+#    priority: 1000
+
+- resource_name: concert_service_gazebo/gazebo_robot_world
+  overrides:
+    parameters: gazebo_concert/gazebo.parameters
```

Comment:
Ahh sorry, I didn't see that you'd given the package the name 'gazebo_concert` in the package.xml [here](https://github.com/tfoote/turtlebot/pull/4/files#diff-e2dcb5a6324bf9a109e2012bada7d4cfR3) 

You chould change that to be the same as the directory name. And not use the same name as another package.